instaclone/src/components/PostSection/postSectionStatus.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {PostStatusContainer} from '../../ComponentStyles/ComponentStyles';
+import './postSection.scss';
+
+class PostSectionStatus extends React.Component{
+
+    state ={
+        likes:this.props.data
+    }
+
+
+    likePost = () =>{
+        const addLike = parseInt(this.state.likes) + 1;
```

Comment:
the logic for liking a post shouldn't be +1 everytime the button is clicked. You either add 1 by clicking, or remove the 1 by clicking it again.